ACE/ace/Log_Msg.cpp

```diff
@@ -753,8 +753,8 @@ ACE_Log_Msg::cleanup_ostream ()
           ACE_OS::fclose (this->ostream_);
 #else
           delete this->ostream_;
-          this->ostream_ = 0;
 #endif
+          this->ostream_ = 0;
```

Comment:
Same for ostream_refcount_? Shouldn’t that also be set to nullptr?

Reply:
> Same for ostream_refcount_? Shouldn’t that also be set to nullptr?

I am not sure. 

In fact, `ostream_refcount_` is set to `nullptr` outiside `if`

Reply:
Ok, missed that, on my phone